airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/staging/StagingConsumerFactory.java

```diff
@@ -213,7 +212,7 @@ private OnCloseFunction onCloseFunction(final JdbcDatabase database,
           }
           queryList.add(stagingOperations.copyTableQuery(database, schemaName, srcTableName, dstTableName));
         }
-
+        stagingOperations.onDestinationCloseOperations(database, writeConfigs.stream().map(WriteConfig::getOutputSchemaName).collect(Collectors.toSet()));
```

Comment:
Not necessary but maybe shift this part `writeConfigs.stream().map(WriteConfig::getOutputSchemaName).collect(Collectors.toSet())` in a separate method with name which describe what we do here

Reply:
The description of this interface is located in the interface by itsself, please see
@suhomud